nordeus/crowd-user-bundle/2.0/config/packages/security.yaml.example

```diff
@@ -0,0 +1,24 @@
+security:
```

Comment:
Maybe rename this file to `nordeus_security.yaml`?

Or just replace it with a link to the docs.